sample-config.yaml

```diff
@@ -3,9 +3,9 @@ kind: "NetworkConfig"
 metadata:
   name: "default"
 spec:
-  serviceNetwork: "172.30.0.0/16"
+  serviceNetwork: "10.3.0.0/16"
   clusterNetworks:
-    - cidr: "10.128.0.0/14"
+    - cidr: "10.2.0.0/16"
```

Comment:
Where did these new defaults come from?

Putting `serviceNetwork` immediately after `clusterNetworks[0].cidr` means that you can't grow that CIDR. I guess since we support multiple clusterNetwork CIDRs now, this isn't fatal, but it's annoying, since it means that if you want to grow from the default, you'll probably end up with, eg, pods on `10.2.0.0/16`, services on `10.3.0.0/16`, and then more pods on `10.4.0.0/16`.

More generically, it is nice to be able to distinguish pod IPs from service IPs at a glance, and using such similar CIDR ranges makes that harder.

If we're changing defaults, we should also consider dropping `hostSubnetLength` back down to 8 since that results in nicer IP range allocations and I think 253 is already more pods-per-node than we support (scaling-wise) on expected cluster configurations anyway.

Also, if we change any of the defaults there will need to be docs updates

Reply:
Good point - I was going off the new installer defaults, but I think those are old Tectonic defaults. I'll change the installer to use the usual Openshift ones, and then update the sample object here (which is just for documentation anyways).